torch/functional.py

```diff
@@ -1235,16 +1235,29 @@ def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa
             None   Frobenius norm                2-norm
             'fro'  Frobenius norm                --
             'nuc'  nuclear norm                  --
-            Other  as vec norm when dim is None  sum(abs(x)**ord)**(1./ord)
+            Other  --                            sum(abs(x)**ord)**(1./ord)
             =====  ============================  ==========================
 
-        dim (int, 2-tuple of ints, 2-list of ints, optional): If it is an int,
-            vector norm will be calculated, if it is 2-tuple of ints, matrix norm
-            will be calculated. If the value is None, matrix norm will be calculated
-            when the input tensor only has two dimensions, vector norm will be
-            calculated when the input tensor only has one dimension. If the input
-            tensor has more than two dimensions, the vector norm will be applied to
-            last dimension.
+            The vector norm can be calculated across any number of dimensions.
+            The corresponding dimensions of :attr:`input` are flattened into
+            one dimension, and the norm is calculated on the flattened
+            dimension. The vector norm will be used if :attr:`p` is a number,
+            or if :attr:`p` is ``None`` and :attr:`input` has either one or
+            more than three dimensions.
+
+            Frobenius norm and nuclear norm can only be calculated across two
+            dimensions.
+
+            When ``p='fro'`` is set, and either :attr:`input` or :attr:`dim`
```

Comment:
Does this statement contradict the description of dim, where it's explained that "If the norm type indicated by :attr:\`p\` does not support the specified number of dimensions, an error will occur."

?